apps/i18n/common/en_us.json

```diff
@@ -1564,6 +1568,10 @@
   "lessonExtrasButton": "Go to Teacher Dashboard",
   "lessonExtrasDetails": "Your students won’t see this page unless you turn them on. You can turn Lesson Extras on by editing section details from your Teacher Dashboard.",
   "lessonMaterials": "Lesson Materials",
+  "theLessonMaterials": "lesson materials",
```

Comment:
Instead of adding this new string, we could use `i18n.lessonMaterials.toLowerCase()`. I'm fine either way, but if we keep this new string, let's rename to make it accurate (there's no 'the')

Reply:
I agree with this. I think we should try to avoid duplicate or near duplicate strings if we can.